layouts/shortcodes/otel-sem-http-table.md

```diff
@@ -0,0 +1,15 @@
+| OpenTelemetry convention | Datadog convention |
+| --- | --- |
+| `client.address` | `http.client_ip` | 
+| `http.response.body.size` | `http.response.content_length` |
+| `http.response.header.<header-name>` | `http.response.headers.<header-name>` |
+| `http.response.status_code` | `http.status_code` | 
+| `http.request.body.size` | `http.request.content_length` |
+| `http.request.header.referrer` | `http.referrer` |
+| `http.request.header.<header-name>` | `http.request.headers.<header-name>` |
+| `http.request.method` | `http.method` |
+| `http.route` | `http.route` |
```

Comment:
i don't see http.route as otel semantics in https://opentelemetry.io/docs/specs/semconv/http/

Reply:
You can see it here https://opentelemetry.io/docs/specs/semconv/attributes-registry/http/